src/lib/core/tests/TestTLV.cpp

```diff
@@ -277,7 +277,7 @@ void TestBufferContents(nlTestSuite * inSuite, const System::PacketBufferHandle
     System::PacketBufferHandle buf = buffer.Retain();
     while (!buf.IsNull())
     {
-        uint16_t len = buf->DataLength();
+        uint32_t len = static_cast<uint32_t>(buf->DataLength());
```

Comment:
Can `len` not be `size_t` here?

Reply:
I think I need to change the passed in argument 'expectedLen' to size_t in order to do that because of all the checks and arithmetic involved in lines below. This seemed like a lesser change and safe as well. But I can check to see if I can change that arg to make len as a size_t.

Reply:
Followup is fine it it's too much work.